dvc/main.py

```diff
@@ -53,6 +54,15 @@ def main(argv=None):
         ret = 253
     except DvcParserError:
         ret = 254
+    except UnicodeError:
+        if is_py2:
+            logger.exception(
+                "unicode is not fully supported in DVC for Python 2.\n"
+                "Python 2.7 will not be maintained past 2020.\n"
```

Comment:
past 2020 means that it will be supported in 2020. Python 2 EOL is Jan 1st 2020, which is not the same.

Reply:
Indeed, I meant `past Jan 1st 2020` :sweat: